filebeat/module/auditd/_meta/fields.yml

```diff
@@ -1,3 +1,120 @@
+- key: user
+  title: "User"
```

Comment:
That this comes first causes `filebeat.reference.yml` to change `Auditd Module` to `User Module`.

Reply:
Ah you're right, I'll fix. This file is tricky.